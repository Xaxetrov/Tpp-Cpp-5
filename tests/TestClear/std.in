S s1 -5 5 -15 -10
R r1 0 0 1 1
PC pc1 0 0 10 0 10 10 0 10
OR or1 s1 pc1
OI oi1 r1 pc1
OR or2 or1 oi1
DELETE r1
CLEAR
LIST
UNDO
LIST
EXIT
