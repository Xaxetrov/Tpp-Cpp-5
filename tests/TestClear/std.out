OK
OK
OK
OK
OK
OK
OK
OK
OK
Intersection named oi1 from 2 objects
Union named or1 from 2 object
Union named or2 from 2 objects
Polygon named pc1
Segment named s1
